ref/appliance-command-reference/gateway.md

```diff
@@ -0,0 +1,49 @@
+The `gateway` command allows and manages access to the HTTP API of the stream gateway. If the HTTP API is exposed (which is the default), authenticated clients can connect to it.
+
+## Usage
+
+    anbox-cloud-appliance gateway <subcommand>
+
+## Subcommands
+
+### `account`
+Manages accounts to access the Anbox stream gateway.
+
+#### Usage
+
+    anbox-cloud-appliance gateway account <subcommand>
+
+#### Subcommands
+
+##### `create`
+Creates an account to access the Anbox stream gateway.
+
+###### Usage
+
+    anbox-cloud-appliance gateway account create <account-name>
+
+This command creates an account with the given <account-name> and returns a token which you can use for accessing the Anbox Stream Gateway HTTP API directly or through the Anbox Stream SDK. See [How to access the stream gateway](https://discourse.ubuntu.com/t/managing-stream-gateway-access/17784) for more information.
+
+##### `delete`
+Deletes the specified account and removes access to the Anbox stream gateway for the specified account.
+
+###### Usage
+
+    anbox-cloud-appliance gateway account delete <account-name>
+
+### `expose`
+Enables access to the stream gateway in the load balancer.
+
+#### Usage
+
+    anbox-cloud-appliance gateway expose
+
+### `unexpose`
+Disables access to the stream gateway in the load balancer.
+
+#### Usage
+
+    anbox-cloud-appliance gateway unexpose
+
+
+
```

Comment:
I'm not sure how we could do this better, but looking at the [rendered file](https://github.com/anbox-cloud/discourse-docs/blob/04aaea81fc3b50d60d386ec2b005c0cf72303d32/ref/appliance-command-reference/gateway.md), I think it's a bit hard to understand at what level a heading is and to what it applies while just reading through it.

It may render differently on our docs website, though.

Reply:
Does it read better after removing the `Usage` heading? If not, perhaps, I can turn the subcommands section into a table format.

Reply:
I think it reads much better, thanks!